scripts/build_scripts/build-adios2-kokkos-frontier.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 # shellcheck disable=SC2191
 
-module load rocm/5.4.0
```

Comment:
craype-accel-amd-gfx90a is needed for the rocm devices to work

Reply:
I'll add it back and see if this is the issue.
It's strange though because I was able to run with HIP on Frontier with just `PrgEnv-gnu-amd/8.5.0`

Reply:
This was not the issue and it doesn't seem we need `craype-accel-amd-gfx90a`
I will look again tonight but I will remove it if all our tests work without it.